### DISCUSSION TITLE
Spectral (Meta)data licenses

### DISCUSSION THREAD

**Discussion by vcnainala** [NFDI4Chem/nmrxiv#123] [Features]
When sharing data, our submitters has to choose a license type for the consumers to be able to use the data. Here are some of the data license types that platforms like Kaggle, ods.ai, DrivenData and others offer. Similarly, we would like to offer our submitters an ability to choose among the following or specify their own license type in the corresponding sections there by enabling total flexibility. 

Creative Commons 
   - CC0:Public Domain - https://creativecommons.org/publicdomain/zero/1.0/
   - CC BY-NC-SA 4.0 - https://creativecommons.org/licenses/by-nc-sa/4.0/
   - CC BY-SA 4.0 - https://creativecommons.org/licenses/by-sa/4.0/
   - CC BY-SA 3.0 - https://creativecommons.org/licenses/by-sa/3.0/
   - Attribution 4.0 International (CC BY 4.0) - https://creativecommons.org/licenses/by/4.0/
   - Attribution-NonCommercial 4.0 International (CC BY-NC 4.0) - https://creativecommons.org/licenses/by-nc/4.0/
   - Attribution 3.0 Unported (CC BY 3.0) - https://creativecommons.org/licenses/by/3.0/
   - Attribution 3.0 IGO (CC BY 3.0 IGO) - https://creativecommons.org/licenses/by/3.0/igo/
   - Attribution-NonCommercial-ShareAlike 3.0 IGO (CC BY-NC-SA 3.0 IGO) - https://creativecommons.org/licenses/by-nc-sa/3.0/igo/
   - Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0) - https://creativecommons.org/licenses/by-nd/4.0/
   - Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) - https://creativecommons.org/licenses/by-nc-nd/4.0/

GPL
   - GPL 2 - http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
   - GNU Lesser General Public License 3.0 - http://www.gnu.org/licenses/lgpl-3.0.html
   - GNU Affero General Public License 3.0 - http://www.gnu.org/licenses/agpl-3.0.html
   - GNU Free Documentation License 1.3 - http://www.gnu.org/licenses/fdl-1.3.html

Open Data Commons
   - Database: Open Database, Contents: Database Contents - http://opendatacommons.org/licenses/dbcl/1.0/
   - Database: Open Database, Contents: © Original Authors - http://opendatacommons.org/licenses/odbl/1.0/
   - ODC Public Domain Dedication and Licence (PDDL) - https://opendatacommons.org/licenses/pddl/1.0/
   - ODC Attribution License (ODC-By) - https://opendatacommons.org/licenses/by/1-0/index.html
 
Community Data License
   - Community Data License Agreement - Permissive - Version 1.0 - https://cdla.io/permissive-1-0/
   - Community Data License Agreement - Sharing - Version 1.0 - https://cdla.io/sharing-1-0/

Special
   - World Bank Dataset Terms of Use - https://www.worldbank.org/en/about/legal/terms-of-use-for-datasets
   - Reddit API Terms - https://www.reddit.com/wiki/api
   - U.S. Government Works - https://www.usa.gov/government-works/
   - EU ODP Legal Notice - https://ec.europa.eu/info/legal-notice_en

Other
   - Data files © Original Authors
   - Other (specified in the description) Unknown

**Comment by NRayya** [comment]
We need a balance between providing more licenses and being less confusing. 
Chemists seem to be not familiar with which licenses when contacted to be asked. Many asked for guidance.
Maybe best option to provide most common licenses, small description when hovering, and a clear link to the documentation with our recommendation provided there clearly. We can recommend CC by 4.0 and highlight why we recommend it. 

For less common licenses, we can add a button for "request another license", which open a window for providing a name and a link, and a message that human curation might take some time and whether they are sure they don't want to use an available license. We can check if this license actually exists and add it to  the project if so.

**Comment by NRayya** [comment]
Here is a list of provided but kind of irrelevant licenses:

- Creative commons licenses v3: v4 is the latest, we should avoid older ones unless requested
-  GPL is a license for Software, not for data: If we insist on having licenses for software, maybe we need to have two fields? to distinguish between the license for the data and the one for the software
- [Open Data Commons Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/) , and [Open Data Commons Attribution License](https://opendatacommons.org/licenses/by/) are for databases. Only [Open Data Commons Public Domain Dedication and License (PDDL)](https://opendatacommons.org/licenses/pddl/) can be used from [Open Data Common](https://opendatacommons.org/)
- [World Bank Dataset Terms of Use](https://www.worldbank.org/en/about/legal/terms-of-use-for-datasets) It is the same as cc by 4.0, but with additional terms to solve disputes, I think it is too much, along with being a license from Bank which feels irrelevant
- Reddit API Terms - https://www.reddit.com/wiki/api are for APIS, they have terms like :All API clients must authenticate with [OAUTH 2](https://github.com/reddit/reddit/wiki/OAuth2)
- U.S. Government Works https://www.usa.gov/government-works/ is for U.S. government creative works
- EU ODP Legal Notice - https://ec.europa.eu/info/legal-notice_en is cc by 4.0

**Comment by NishaSharma14** [comment]
Only 3 in chemotion: CC-0, CC-BY, CC-BY-SA

**Comment by tilfischer** [comment]
For comparison, the licenses in RADAR for research data are CC including PDM1, ODC (the tree mentioned by Noura).

There is a separate list for software licenses, as RADAR can be also a repository for software, while nmrXiv as a field-specific repository is made for NMR (research) data. 

Finally, there is "all right reserved" chooseable and "other", which gives a free text field to specify the license. 

We probably should not include the later two ones as these are unFAIR and free text descriptions are most commonly very painful at some point and also can be unFAIR. Moreover, we should try not to make things more complicated to researchers than needed and researchers are possibly most familiar with CC licenses for their research data. 

Hence, I would suggest to include the CC licenses 4.0 and maybe, if some of you things that this has some benefits, ODC licenses such as the [Open Data Commons Public Domain Dedication and License (PDDL)](https://opendatacommons.org/licenses/pddl/) mentioned by Noura and the [Open Data Commons Attribution License](https://opendatacommons.org/licenses/by/), which can be used for data or databases and was also previously mentioned by Noura. Though, I currently do not see any benefit from adding these licenses.

To make life even more easier for researchers while they are adding their NMR data to nmrXiv, we could also go for a pre-picked licenses, which could be CC BY 4.0. 

Chemotion has CC BY-SA 4.0 as pre-chosen license, while CC BY 4.0 is more than sufficient and the need for attribution should not be confused with the need to cite scientific work in scientific publications, as required e.g. by the DFG Codex.

**Comment by NRayya** [comment]
Additionally we have two licences: "Data files © Original Authors" and "Other (specified in the description) Unknown" where we don't provide a link or description of the licence. They are confusing for re-users and only reusable after contacting the authors, which is not always possible if the authors don't provide emails. We need to sort them out.

**Comment by tilfischer** [comment]
"Data are available on request from the authors" is not just problematic because it might be difficult to reach the authors in the first place but should be avoided, as the probability to actually get the data drops by 17% per year see https://doi.org/10.1016/j.cub.2013.11.014 .
